rdf_differ/config.py

```diff
@@ -19,7 +19,7 @@
         and any(Path(os.environ.get('RDF_DIFFER_TEMPLATE_LOCATION')).iterdir()):
     RDF_DIFFER_REPORT_TEMPLATE_LOCATION = os.environ.get('RDF_DIFFER_TEMPLATE_LOCATION')
 else:
-    RDF_DIFFER_REPORT_TEMPLATE_LOCATION = Path(__file__).parents[1] / 'resources/templates/diff_report'
+    RDF_DIFFER_REPORT_TEMPLATE_LOCATION = str(Path(__file__).parents[1] / 'resources/templates/diff_report')
```

Comment:
why str; Path should normally be preferred

Reply:
it's a safer way to "store" the location, and will be consistent with line `20` where the location will be retrieved as a `string` variable